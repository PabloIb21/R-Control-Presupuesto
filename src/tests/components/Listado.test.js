import React from 'react';
import { mount } from 'enzyme';
import Listado from '../../components/Listado';

describe('<Listado />', () => {
  test('mount', () => {
    const gastos = [
      {
        id: 1,
        nombre: 'Transporte',
        cantidad: 300
      },
      {
        id: 2,
        nombre: 'Comida',
        cantidad: 200
      }
    ];

    const wrapper = mount(
      <Listado gastos={gastos} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});