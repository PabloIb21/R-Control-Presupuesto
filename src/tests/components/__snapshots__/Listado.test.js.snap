// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Listado /> mount 1`] = `
<div
  className="gastos-realizados"
>
  <h2>
    Listado
  </h2>
  <li
    className="gastos"
  >
    <p>
      Transporte
      <span
        className="gasto"
      >
        $ 
        300
      </span>
    </p>
  </li>
  <li
    className="gastos"
  >
    <p>
      Comida
      <span
        className="gasto"
      >
        $ 
        200
      </span>
    </p>
  </li>
</div>
`;
